src/main/java/ch/njol/skript/expressions/ExprHealAmount.java

```diff
@@ -83,42 +84,48 @@ public Class<?>[] acceptChange(Changer.ChangeMode mode) {
 			return null;
 		return CollectionUtils.array(Number.class);
 	}
-	
+
 	@Override
-	public void change(Event e, @Nullable Object[] delta, Changer.ChangeMode mode) {
-		if (!(e instanceof EntityRegainHealthEvent))
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
+		if (!(event instanceof EntityRegainHealthEvent))
 			return;
 
+		EntityRegainHealthEvent healthEvent = (EntityRegainHealthEvent) event;
 		double value = delta == null ? 0 : ((Number) delta[0]).doubleValue();
 		switch (mode) {
 			case SET:
 			case DELETE:
-				((EntityRegainHealthEvent) e).setAmount(value);
+				healthEvent.setAmount(value);
 				break;
 			case ADD:
-				((EntityRegainHealthEvent) e).setAmount(((EntityRegainHealthEvent) e).getAmount() + value);
+				healthEvent.setAmount(healthEvent.getAmount() + value);
 				break;
 			case REMOVE:
-				((EntityRegainHealthEvent) e).setAmount(((EntityRegainHealthEvent) e).getAmount() - value);
+				healthEvent.setAmount(healthEvent.getAmount() - value);
 				break;
 			default:
 				break;
 		}
 	}
-	
+
+	@Override
+	public boolean setTime(int time) {
+		return super.setTime(time);
+	}
```

Comment:
What's the point of this?

Reply:
For proper grammar support. `if the healing amount will be greater than 50`

Reply:
I just don't understand why it overrides? Not overriding it would be the exact same behavior? I don't see why it needs to explicitly override the super method if all it does is just call the super method anyway.